Walnut/src/Walnut/EntryPoint.h

```diff
@@ -1,7 +1,5 @@
 #pragma once
 
-#ifdef WL_PLATFORM_WINDOWS
```

Comment:
This is good that this preprocessor check is removed. In the case that WL_DIST is defined for a release-distribution build, Windows only code will be admitted below!